hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DatasourceJBossASClient.java

```diff
@@ -152,33 +152,46 @@ public boolean isXADatasource(String datasourceName) throws Exception {
     /**
      * @param driverName the name of the JDBC driver to create, e.g. {@code "h2"}
      * @param moduleName the name of the WIldFly module that contains the driver jar file
+     * @return
```

Comment:
empty return javadoc will fail javadoc verifier (if we ever run that). just put something in there after "return" to minimally describe it